assets/sass/layout/_grid.scss

```diff
@@ -87,11 +87,11 @@
 }
 
 .grid-secondary-column__item--wide-only {
+  display: none;
 
-  @media only all and (max-width: #{get-rem-from-px($bkpt-site--wide - 1)}em) {
-    display: none;
+  @media only screen and (min-width: #{get-rem-from-px($bkpt-site--wide)}em) {
+    display: block;
```

Comment:
This is the only change that needs scrutiny really. What is the reason for this as this class is used in journal.

See: https://github.com/elifesciences/journal/blob/986eb9b8197f8dbe0a17e0ef0ea5877dce70c0f5/app/Resources/views/article-text.html.twig#L49

Can you explain why this change is needed and what effect it will have on journal?

All other changes. My only suggestion is that we could also add an example of a related item with a see more link as this is quite common on the journal article pages.

If @chris-huggins is happy that every is as expected on the affected pages then this can be merged.

Reply:
> This is the only change that needs scrutiny really. What is the reason for this as this class is used in journal.

The class "grid-secondary-column__item--wide-only" is used only with Related items and contains style to hide the content until 899px and to show it above 900px. When we are using max-width to apply for all screens sizes until 900px - 1, converting to rems doesn't work for 899px, I tried to avoid it but I also can see there is a lot of max-width using. If it's not an issue, I can revert it.

Reply:
I created new example only for research article pages with related item and see more link.